synapse/server.py

```diff
@@ -186,6 +190,9 @@ def setup(self):
         self.datastore = DataStore(self.get_db_conn(), self)
         logger.info("Finished setting up.")
 
+    def get_reactor(self):
```

Comment:
I really wouldn't mind if this had a docstring; "Returns the twisted reactor in use by this HomeServer" or something?